### DISCUSSION TITLE
Firefox development

### DISCUSSION THREAD

**Discussion by MEGA-Dawg68** [SethBurkart123/EvenBetterSEQTA#174] [General]
How are we developing the firefox branch? I see it has been updated 9 months ago but it has the latest features in firefox. I have an issue and I want to try and fix it. So where would I actually find the firefox code?

**Comment by Crazypersonalph** [comment]
Firefox code is now the same as the chrome code and other browsers, as we have fully implemented polyfilling and cross-platform tests.

The firefix-dev branch is irrelevant
